src/pages/Phase_04.svelte

```diff
@@ -10,25 +10,72 @@ Trustee: how angry/satisfied
 Investor: how guilty/satisfied
 
 Data needed:
-- Investor expectation from phase 2 -> DB
-- Trustee expectation from phase 2 -> DB
-- Investor earned from phase 3 -> DB
-- Trustee earned from phase 3 -> DB
+- [x] Investor expectation from phase 2 -> DB
+- [x] Trustee expectation from phase 2 -> DB
+- [x] Investor earned from phase 3 -> DB
+- [x] Trustee earned from phase 3 -> DB
 
 Data stored/modified:
 - Investor anger/satisfcation
 - Trustee guilt/satisfcation
 -->
 <script>
   import { createEventDispatcher } from "svelte";
-  import { userStore, groupStore, saveAPQData, globalVars } from "../utils.js";
+  import {
+    userStore,
+    groupStore,
+    saveAPQData,
+    globalVars,
+    saveQData,
+    round2,
+  } from "../utils.js";
   import Loading from "../components/Loading.svelte";
   import MouseRating from "../components/MouseRating.svelte";
   import Button from "../components/Button.svelte";
 
   const dispatch = createEventDispatcher();
   let submitted = false;
 
+  // GET TRIAL DATA
+  let currentTrial = $groupStore.currentTrial;
+  let totalTrials = $groupStore.trials.length;
+
+  // Shared endowment
+  let endowment = $groupStore.trials[currentTrial].endowment;
+  let I_expectation = $groupStore.trials[currentTrial].I_1ST_ORDER_EXPECTATION;
+  let T_prediction = $groupStore.trials[currentTrial].T_PREDICTION;
+  let I_earnings = $groupStore.trials[currentTrial].I_EARNED;
+  let T_earnings = $groupStore.trials[currentTrial].T_EARNED;
+  let I_received = $groupStore.trials[currentTrial].T_CHOICE;
+  let T_received = $groupStore.trials[currentTrial].I_CHOICE;
+
+  const earnings = round2(
+    $userStore.role === "investor" ? I_earnings : T_earnings
+  );
+  const expectFromOther =
+    $userStore.role === "investor" ? I_expectation : T_prediction;
+  const receivedAmount =
+    $userStore.role === "investor" ? I_received : T_received;
+  const keptAmount = round2(
+    $userStore.role === "investor" ? endowment - T_received : T_received
+  );
+  const otherName =
+    $userStore.role === "investor" ? $groupStore.T_name : $groupStore.I_name;
+
+  const outcomeExplanation =
```

Comment:
ooo I really like this. One enhancement could be maybe like color coding each piece of info? Something like "Eshin *started* with..", where *started* is 1 color. Then "You *expected*..." where, *expected* is a different color, etc etc. Might make it easier for them to parse mutliple lines of text quickly.

Reply:
cdd907bdf10c0f449c4eafd0957089ca7b2efb9a

Reply:
OK, in commit (cdd907bdf10c0f449c4eafd0957089ca7b2efb9a), color-coded some text. Doesn't feel quite complete yet; there may be other things that should be color-coded or emphasized in a certain way, but at least there's now an `outcomeExplanation` object with fields that can be stylized/color-coded with span classes... Let me know if there are ways to better this. This is just what I remembered from what we discussed